include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -0,0 +1,112 @@
+//
+// Created by Jan Drewniok on 23.11.22.
+//
+
+#ifndef FICTION_CHARGE_DISTRIBUTION_SURFACE_HPP
+#define FICTION_CHARGE_DISTRIBUTION_SURFACE_HPP
+
+#include "fiction/technology/cell_technologies.hpp"
+#include "fiction/technology/sidb_defects.hpp"
+#include "fiction/traits.hpp"
+
+#include <unordered_map>
+
+namespace fiction
+{
+
+/**
+ * Possible SiDB charges.
+ */
+enum class sidb_charge_states
+{
+    NONE,  // assigned when layout cell is empty
+    POSITIVE,
+    NEUTRAL,
+    NEGATIVE
+};
+
+struct sidb_charge
```

Comment:
We don't need this struct. In `charge_distribution_surface`, we can map directly from `coordinate` to `sidb_charge_state`